lib/Config/Hooks.php

```diff
@@ -166,6 +177,21 @@ function setupSubscriptionEvents() {
     );
   }
 
+  function setupMailer() {
+    $this->wp->addAction('plugins_loaded', [
```

Comment:
I am not sure, I used the example from the specification: https://plugins.trac.wordpress.org/browser/wp-mail-smtp/trunk/src/Core.php#L543

Reply:
I was looking where $phpmailer variable is assigned a value in WP and found only [one place](https://core.trac.wordpress.org/browser/tags/5.2/src/wp-includes/pluggable.php#L215), and it tests if the variable contains the mailer class. So I would say that it should be possible to define it.

Maybe we could use both:
1) Define our mailer during initialization - this could cover all earlier events like user registration email etc.
2) Do it after plugins initialization to ensure that if some other plugin replaces it as we would do in 1) we replace it again.